docs/developer/extending/webhooks/overview.mdx

```diff
@@ -87,3 +87,15 @@ Saleor can send Webhooks for various events that happen in the system. Events ca
 The main and **most important difference** between synchronous and asynchronous events is that **synchronous events execution influences the response time of the request that triggered the event**. That said if you have a request that triggers a synchronous event and the event processing takes a long time, the request will take a long time to finish as well affecting the response time of the request.
 
 You can read more about asynchronous events in the [Asynchronous events](./asynchronous-events.mdx) section and about synchronous events in the [Synchronous events](./synchronous-events/overview.mdx) section.
+
+### Requests time
+
+Both synchronous and asynchronous webhook requests are time-limited. Saleor will wait a maximum of 20 seconds for the full HTTP request round trip, which leaves approximately 18 seconds to process the data and respond.
+
+Some operations can contain more than one synchronous webhook invocation. Saleor can't guarantee successful processing if a summary of the entire GraphQL request exceeds 30 seconds.
+
+You can follow some good practices to avoid webhook timeouts:
+
+- For asynchronous webhooks use job queues like SQS. "Accept" response from the queue is fast and the rest of the processing should be managed on the external system like an app.
```

Comment:
```suggestion
- Use queues instead of HTTP endpoints for asynchronous events. Saleor will send events straight to the queue, and your application can process them at its own pace without blocking Saleor.
```